src/biotite/structure/graphics/rna.py

```diff
@@ -73,9 +73,11 @@ def plot_nucleotide_secondary_structure(
         The *Matplotlib* compatible linestyle of the backbone.
     backbone_color : str or ndarray, shape=(3,) or shape=(4,), dtype=float, optional (default: 'grey')
         The *Matplotlib* compatible color of the backbone.
-    base_font : dict, optional (default: {'size': 'smaller'})
-        The *Matplotlib* compatible font of the labels denoting the type
-        of each base.
+    base_text : dict or iterable, optional (default: {'size': 'smaller'})
+        The *Matplotlib* attributes of the labels denoting the type of 
```

Comment:
I would replace '\*Matplotlib\* attributes' with 'keyword parameters for the \*Matplotlib\* \`Text\` objects', to clarify the meaning